--- work/configure.py.orig	2017-09-05 21:10:09.476485425 +0000
+++ work/configure.py	2017-09-05 21:10:20.692482113 +0000
@@ -2346,7 +2346,7 @@
     mk_clean_dir(mname)
 
     # Build the SIP command line.
-    argv = [quote(target_config.sip), '-w', '-f', sip_flags]
+    argv = [quote(target_config.sip), '-w', sip_flags]
 
     # Make sure any unknown Qt version gets treated as the latest Qt v5.
     argv.append('-B')
