--- configure.py.orig	2015-06-12 01:28:16.000000000 +1000
+++ configure.py	2017-04-12 18:46:58.840990745 +1000
@@ -49,7 +49,8 @@
 
 # This default value will be used by the code that bootstraps the Qt
 # configuration.
-qt_macx_spec = 'macx-g++'
+# Needs not to be g++ on Travis since compilation of tests fail
+qt_macx_spec = os.environ["QMAKESPEC"]
 
 qt_sip_flags = []
 
@@ -1854,6 +1855,15 @@
     if macros is None:
         return None
 
+    # QMake macros may contain comments on the same line so we need to remove them
+    if sys.version[0] == '3':
+        mitems = list(macros.items())
+    else:
+        mitems = macros.iteritems()
+    for macro, value in mitems:
+        if "#" in value:
+            macros[macro] = value.split("#", 1)[0]
+
     # Qt5 doesn't seem to support the specific macros so add them if they are
     # missing.
     if macros.get("INCDIR_QT", "") == "":
