diff --git a/home/leofang/sources/PyQt5_gpl-5.12.3/configure-original.py b/home/leofang/sources/PyQt5_gpl-5.12.3/configure.py
index d85f778..c0dbc4f 100644
--- a/home/leofang/sources/PyQt5_gpl-5.12.3/configure-original.py
+++ b/home/leofang/sources/PyQt5_gpl-5.12.3/configure.py
@@ -803,19 +803,6 @@ class TargetConfiguration:
         configuration.  opts are the command line options.
         """
 
-        # On Windows the interpreter must be a debug build if a debug version
-        # is to be built and vice versa.
-        if sys.platform == 'win32':
-            if opts.debug:
-                if not self.py_debug:
-                    error(
-                            "A debug version of Python must be used when "
-                            "--debug is specified.")
-            elif self.py_debug:
-                error(
-                        "--debug must be specified when a debug version of "
-                        "Python is used.")
-
         self.debug = opts.debug
 
         # Get the target Python version.
@@ -2078,6 +2065,12 @@ def run_qmake(target_config, verbose, pro_name, makefile_name='', fatal=True, re
 
         return False
 
+    with open(mf, 'r') as f:
+        data = f.read()
+        print("~~~~~~ DEBUG ~~~~~~")
+        print(data)
+        print("~~~~~~~~~~~~~~~~~~~")
+
     # Restore the current directory.
     if cwd is not None:
         os.chdir(cwd)
@@ -2107,6 +2100,7 @@ def run_make(target_config, verbose, exe, makefile_name):
         else:
             makefile_target = 'release'
             platform_exe = os.path.join('release', exe + '.exe')
+        sys.stdout.write("xxxxx DEBUG: " + platform_exe + '\n')
     else:
         make = 'make'
         makefile_target = ''
@@ -2123,6 +2117,7 @@ def run_make(target_config, verbose, exe, makefile_name):
     if makefile_target != '':
         args.append(makefile_target)
 
+    sys.stdout.write("ooooo DEBUG: " + ' '.join(args) + '\n')
     run_command(' '.join(args), verbose)
 
     return platform_exe if os.access(platform_exe, os.X_OK) else None
@@ -2134,7 +2129,7 @@ def run_command(cmd, verbose):
     """
 
     if verbose:
-        sys.stdout.write(cmd + "\n")
+        sys.stdout.write("***** DEBUG: " + cmd + "\n")
 
     fout = get_command_output(cmd, and_stderr=True)
 
