--- PyQt5_gpl-5.12.3/qpy/QtCore/qpycore_qstring.cpp	2019-06-25 12:41:02.000000000 +0000
+++ PyQt5_gpl-5.13.1/qpy/QtCore/qpycore_qstring.cpp	2019-09-07 19:37:19.000000000 +0000
@@ -158,9 +158,21 @@
 }
 
 
-// Convert a Python Unicode object to a QString.
+// Convert a Python string object to a QString.
 QString qpycore_PyObject_AsQString(PyObject *obj)
 {
+#if PY_MAJOR_VERSION < 3
+    if (PyString_Check(obj))
+    {
+        const char *obj_s = PyString_AsString(obj);
+
+        if (!obj_s)
+            return QString();
+
+        return QString::fromUtf8(obj_s);
+    }
+#endif
+
 #if defined(PYQT_PEP_393)
     int char_size;
     Py_ssize_t len;
