diff --git a/configure.py b/configure.py
index 35fc985..3d41eab 100644
--- a/configure.py
+++ b/configure.py
@@ -20,7 +20,8 @@ import sys
 import os
 import glob
 import optparse
-from distutils import sysconfig
+import distutils.sysconfig
+import sysconfig
 
 try:
     from importlib import invalidate_caches
@@ -219,9 +220,9 @@ def set_platform_directories():
     global plat_py_conf_inc_dir, plat_bin_dir, plat_py_lib_dir, plat_sip_dir
 
     # We trust distutils for some stuff.
-    plat_py_site_dir = sysconfig.get_python_lib(plat_specific=1)
-    plat_py_inc_dir = sysconfig.get_python_inc()
-    plat_py_venv_inc_dir = sysconfig.get_python_inc(prefix=sys.prefix)
+    plat_py_site_dir = distutils.sysconfig.get_python_lib(plat_specific=1)
+    plat_py_inc_dir = distutils.sysconfig.get_python_inc()
+    plat_py_venv_inc_dir = distutils.sysconfig.get_python_inc(prefix=sys.prefix)
     plat_py_conf_inc_dir = os.path.dirname(sysconfig.get_config_h_filename())
 
     if sys.platform == "win32":
@@ -248,7 +249,7 @@ def set_platform_directories():
         plat_bin_dir = bin_dir
         plat_sip_dir = sys.prefix + "\\sip"
     else:
-        lib_dir = sysconfig.get_python_lib(plat_specific=1, standard_lib=1)
+        lib_dir = distutils.sysconfig.get_python_lib(plat_specific=1, standard_lib=1)
 
         plat_py_lib_dir = lib_dir + "/config"
         plat_bin_dir = sys.exec_prefix + "/bin"
